app/Http/Controllers/CampaignController.php

```diff
@@ -5,19 +5,23 @@
 use App\Http\Requests\CampaignStoreRequest;
 use App\Http\Requests\CampaignUpdateRequest;
 use App\Models\Campaign;
+use App\Models\User;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Inertia\Inertia;
 
 class CampaignController extends Controller
 {
     /**
      * @param \Illuminate\Http\Request $request
-     * @return \Illuminate\Http\Response
+     * @return \Inertia\Response
      */
     public function index(Request $request)
     {
-        $campaigns = Campaign::all();
+        $user = User::where('id', Auth::user()->id)->get();
+        $userCampaigns = $user->first()->campaigns()->get();
```

Comment:
this can just be
```suggestion
        $campaigns = Auth::user()->campagins
```

Reply:
I'll change it, good refactor opportunity!